android/res/layout/fragment_transfer_detail_details.xml

```diff
@@ -0,0 +1,794 @@
+                android:contentDescription="@null"/>
+
+        </RelativeLayout>
+
+        <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="@drawable/divider_line"
+        android:contentDescription="@null"/>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:paddingBottom="@dimen/transfers_detail_page_padding_vertical"
+            android:paddingEnd="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingLeft="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingRight="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingStart="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingTop="@dimen/transfers_detail_page_padding_vertical">
+
+            <TextView
+                style="@style/transferDetailTitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:text="@string/created_on" />
+
+            <TextView
+                style="@style/transferDetailText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/transfer_detail_date_dummy" />
+        </LinearLayout>
+
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@drawable/divider_line"
+            android:contentDescription="@null"/>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingBottom="@dimen/transfers_detail_page_padding_vertical"
+            android:paddingEnd="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingLeft="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingRight="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingStart="@dimen/transfers_detail_page_padding_horizontal"
+            android:paddingTop="@dimen/transfers_detail_page_padding_vertical">
+
+            <TextView
+                style="@style/transferDetailTitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingBottom="5dp"
+                android:text="@string/comment" />
+
+            <TextView
+                style="@style/transferDetailText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/transfer_detail_comment_dummy" />
+        </LinearLayout>
+    </LinearLayout>
+</ScrollView>
+
```

Comment:
that's one long layout... hmm

Reply:
As per my comment above "too much nesting on one of the fragments" - so I was aware and I would like to later populate many of those programatically. This was just in the beginning to show you how the info would look. Lets go over all the torrent info you would like to include and finalize the design first - I will start on making all the necessary changes right after.